flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/ConfigVariablesManager.java

```diff
@@ -123,6 +129,37 @@ private Property property(String label, String description, Property.ValueType v
         return propertyBuilder.build();
     }
 
+    public JsonElement update(Document document, Path configFile, JsonElement configs) {
+        List<ConfigVariable> configVariables = gson.fromJson(configs, ConfigVariables.class).configVariables();
+        StringBuilder sb = new StringBuilder();
+        for (ConfigVariable configVariable : configVariables) {
+            Map<String, Property> properties = configVariable.properties();
+            String value = properties.get(DEFAULTABLE).toSourceCode();
```

Comment:
If the value is missing, it may lead to a null pointer exception. A better approach is to throw a specific exception, making it easier for the frontend to identify the exact issue.

We have addressed similar cases in the `toSourceCode` method of any flow node.

Reply:
Changed